analysis/package.json

```diff
@@ -13,7 +13,7 @@
     "hydrolysis": "1.25.0",
     "lockfile": "1.0.1",
     "npm": "^5.0.3",
-    "polymer-analyzer": "^2.7.0",
+    "polymer-analyzer": "3.0.0-pre.10",
```

Comment:
11 was just released, might as well bump it (probably no breaking changes for you https://github.com/Polymer/polymer-analyzer/blob/master/CHANGELOG.md)